README.md

```diff
@@ -1,77 +1,373 @@
+#### Change Working Directory
 
+Now that we have made a new directory lets go there where we will put our work on our machine. te do that by in both windows and MacOS we do this by dyping cd then pressing enter:
 
+`cd VetsWhoCode`
 
-[Javascript Info](http://javascript.info/)
 
+#### Change to previous Directory
+If you want to go back to the previous directory for both Windows and MacOS you type this and press enter:
 
+`cd ..`
 
-<h2 id="capstone">Capstone</h2>
+#### Delete a directory
+Let's say you make a typo and need to delete a directory. Here are the commands to do that.
 
-Build a website telling about you and include your favorite things and then push it to GitHub. Bonus points if you use [GitHub Pages](https://pages.github.com/) or [surge.sh](https://surge.sh/) to host your site.
+MacOS; `rm -r NameOfDirectory`
+Windows: `rmdir /S NameOfDirectory`
+
+### Code Editor
+
+An Integrated Development Environment (IDE) is a software application that provides comprehensive tools and features to assist developers in writing, debugging, and deploying software. An IDE typically includes a code editor, a debugger, a compiler or interpreter, and other tools such as a profiler, a version control system, and a build automation tool.
+An IDE can also provide features like auto-completion, syntax highlighting, and code refactoring to make coding faster and more efficient. IDEs are commonly used for developing software applications, web applications, and mobile apps. Our preferred IDE is Visual Studio Code. Download it [here](https://code.visualstudio.com/). You can also install our developer extension pack by going to extensions and typing [VetsWhoCode Extension Pack](https://marketplace.visualstudio.com/items?itemName=VetsWhoCode.vetswhocode-extension-pack).
+
+### Browser
+
+The preferred browser for Vets Who Code is the Google Chrome, due to its popularity and its robust developer tools. To download it please go [here](https://www.google.com/chrome/) and follow the prompts.
+
+
+### Using GitHub
+ We use GitHub extensively here. Below you’ll find the steps to use it within the terminal or command prompt.
+
+1.	 Install Git: Git is a version control system used by GitHub. You can download and install Git for Windows from the official Git website.
+2.	 Configure Git: After installing Git, open the Git Bash application and configure your name and email address using the following commands:
+```console
+git config --global user.name "Your Name"
+git config --global user.email "youremail@example.com"
+```
+
+
+
+3.	Create a repository: A repository is a place where you can store and manage your code. Click on the “New” button and create a new repository.
+4.	Clone the repository: To start working with the repository, you need to clone it to your local machine. You can do this by clicking on the “Code” button and copying the repository URL. Then, open your terminal or command prompt and type `git clone URL-Goes-Here`.
```

Comment:
"Clone" is a new term at this point. I retitle this step to `Download the repository` and explain: To start working with the repository, you need to download it to your local machine. Github calls this cloning`